internal/config/config.go

```diff
@@ -121,18 +123,6 @@ func Get() Config {
 		}
 	}
 
-	if cfg.GKE != nil {
-		if cfg.GKE.Region == "" {
-			requiredWhenDiscoveryDisabled("GKE_REGION")
-		}
-		if cfg.GKE.ProjectID == "" {
-			requiredWhenDiscoveryDisabled("GKE_PROJECT_ID")
-		}
-		if cfg.GKE.ClusterName == "" {
-			requiredWhenDiscoveryDisabled("GKE_CLUSTER_NAME")
-		}
-	}
```

Comment:
These should be kept but updated to match the current env variable requirements. There will always be cases where it's not possible to discover something via metadata (blocked), so we suggest users to add the env vars themselves. The env var validation is useful because they will immediately get feedback without our intervention if they get the env variables wrong.

Reply:
Disregard, you added the messages to the gke client.

Reply:
In general when ENV variables are provided they override any autodiscovery. This is the behaviour here as well.
If ENV variable is not provided -> autodiscovery kicks in. If autodiscovery fails it will print an error with ENV variable information to resolve the issue.